tests/builder/instances_test.py

```diff
@@ -45,26 +45,35 @@ def test_apply_instance_data(tmpdir, instance_names):
     designspace = glyphsLib.to_designspace(font, instance_dir=instance_dir)
     path = str(tmpdir / (font.familyName + '.designspace'))
     write_designspace_and_UFOs(designspace, path)
-    builder = DesignSpaceDocumentReader(designspace.path, ufoVersion=3)
+
+    test_designspace = DesignSpaceDocument()
+    test_designspace.read(designspace.path)
     if instance_names is None:
-        # generate all instances
-        builder.process()
-        include_filenames = None
+        # Generate all instances.
```

Comment:
hm the comment says "generate all instances" but we aren't calling mutatormath any more.. where are these UFO instances coming from now? you sure this test code is even running?

Reply:
All tests pass:
```
$ tox -e py36 -- -k apply_instance_data
[...]
tests/builder/instances_test.py::test_apply_instance_data[default] PASSED [ 33%]
tests/builder/instances_test.py::test_apply_instance_data[include_1] PASSED [ 66%]
tests/builder/instances_test.py::test_apply_instance_data[include_2] PASSED [100%]
```
I think the comment is wrong, all the code does is iterate over the instances defined in the generated designspace file.